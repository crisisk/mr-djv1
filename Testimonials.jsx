import React from 'react'
import { useTranslation } from 'react-i18next'

const StarRating = ({ rating }) => {
  const stars = Array(5)
    .fill(0)
    .map((_, i) => (
      <svg
        key={i}
        className={`w-5 h-5 ${i < rating ? 'text-secondary' : 'text-neutral-gray-500'}`}
        fill="currentColor"
        viewBox="0 0 20 20"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path d="M9.049 2.927c.3-.921 1.603-.921 1.902 0l1.07 3.292a1 1 0 00.95.69h3.462c.969 0 1.371 1.24.588 1.81l-2.8 2.034a1 1 0 00-.364 1.118l1.07 3.292c.3.921-.755 1.688-1.54 1.118l-2.8-2.034a1 1 0 00-1.175 0l-2.8 2.034c-.784.57-1.838-.197-1.539-1.118l1.07-3.292a1 1 0 00-.364-1.118l-2.8-2.034c-.783-.57-.38-1.81.588-1.81h3.461a1 1 0 00.951-.69l1.07-3.292z" />
      </svg>
    ))
  return <div className="flex">{stars}</div>
}

const TestimonialCard = ({ testimonial }) => {
  return (
    <div className="bg-neutral-light p-spacing-xl rounded-lg shadow-xl flex flex-col h-full">
      <StarRating rating={testimonial.rating} />
      <p className="text-font-size-h3 text-neutral-dark italic my-spacing-lg flex-grow">"{testimonial.quote}"</p>
      <div className="border-t border-neutral-gray-100 pt-spacing-md">
        <p className="text-font-size-body font-bold text-primary">{testimonial.author}</p>
        <p className="text-font-size-small text-neutral-gray-500">{testimonial.source}</p>
      </div>
    </div>
  )
}

const InitialsAvatar = ({ initials, className = '' }) => (
    <div
        className={`w-12 h-12 rounded-full bg-gradient-to-r from-primary to-secondary text-white font-bold flex items-center justify-center ${className}`}
        aria-hidden="true"
    >
        {initials}
    </div>
);

const TestimonialCard = ({ testimonial }) => {
    return (
        <article className="bg-white border-2 border-neutral-gray-100 rounded-xl p-spacing-xl shadow-sm transition-transform duration-300 hover:-translate-y-1 hover:shadow-lg" aria-label={`Testimonial van ${testimonial.name}`}>
            <header className="flex items-start justify-between gap-spacing-md mb-spacing-lg">
                <div className="flex items-center gap-spacing-md">
                    <InitialsAvatar initials={testimonial.initials} className="w-11 h-11 text-font-size-h5" />
                    <div>
                        <p className="text-font-size-body font-extrabold text-neutral-dark leading-tight">{testimonial.name}</p>
                        <p className="text-font-size-small text-neutral-gray-500">{testimonial.event}</p>
                    </div>
                </div>
                <StarRating rating={testimonial.rating} label={`${testimonial.rating} van 5 sterren`} />
            </header>
            <p className="text-font-size-body text-neutral-gray-700 italic mb-spacing-lg">“{testimonial.quote}”</p>
            <footer className="flex items-center justify-between border-t border-neutral-gray-100 pt-spacing-md text-font-size-small text-neutral-gray-500">
                <time dateTime={testimonial.dateISO}>{testimonial.date}</time>
                <span className="flex items-center gap-1 font-semibold text-primary">
                    <span aria-hidden="true">{testimonial.platform.icon}</span>
                    {testimonial.platform.label}
                </span>
            </footer>
        </article>
    );
};

const StatsSection = () => (
    <section className="bg-neutral-light border-2 border-primary rounded-xl p-spacing-xl grid grid-cols-2 md:grid-cols-4 gap-spacing-lg" aria-label="Service statistieken">
        {statsData.map((stat) => (
            <div key={stat.id} className="relative text-center">
                <VisuallyHidden>{stat.srLabel}</VisuallyHidden>
                {stat.highlight ? (
                    <p className="text-font-size-h4 text-secondary mb-spacing-xs" aria-hidden="true">
                        {stat.highlight}
                    </p>
                ) : null}
                <p className="text-font-size-h2 font-black text-neutral-dark">{stat.value}</p>
                <p className="text-font-size-small font-semibold text-neutral-gray-600 uppercase tracking-wide">
                    {stat.label}
                </p>
            </div>
        ))}
    </section>
);

const FeaturedTestimonial = () => (
    <article className="relative overflow-hidden rounded-2xl bg-gradient-to-r from-primary to-neutral-dark text-white p-spacing-2xl" aria-label={`Uitgelichte testimonial van ${featuredTestimonial.author}`}>
        <div className="absolute inset-y-0 right-0 w-64 bg-gradient-to-b from-secondary/30 to-transparent opacity-50 pointer-events-none" aria-hidden="true" />
        <div className="relative z-10 flex flex-col gap-spacing-xl md:flex-row md:items-start md:justify-between">
            <div className="md:max-w-2xl">
                <p className="inline-flex items-center gap-2 rounded-full bg-secondary/20 px-spacing-md py-spacing-xs text-font-size-small font-bold text-secondary">
                    {featuredTestimonial.badge}
                </p>
                <p className="mt-spacing-lg text-font-size-h4 font-semibold leading-relaxed italic">
                    “{featuredTestimonial.quote}”
                </p>
                <div className="mt-spacing-xl flex items-center gap-spacing-md">
                    <InitialsAvatar initials={featuredTestimonial.initials} className="w-12 h-12 text-font-size-h4" />
                    <div>
                        <p className="text-font-size-body font-extrabold">{featuredTestimonial.author}</p>
                        <p className="text-font-size-small text-neutral-gray-100">{featuredTestimonial.event}</p>
                    </div>
                </div>
            </div>
            <div className="flex flex-col items-center justify-center gap-spacing-md text-center md:text-right">
                <StarRating rating={featuredTestimonial.rating} label={`${featuredTestimonial.rating} van 5 sterren`} />
                <p className="text-font-size-body font-semibold text-secondary">{featuredTestimonial.ratingText}</p>
                <p className="flex items-center gap-2 rounded-lg bg-white/10 px-spacing-md py-spacing-xs text-font-size-small font-semibold">
                    <span aria-hidden="true">{featuredTestimonial.platform.icon}</span>
                    {featuredTestimonial.platform.label}
                </p>
            </div>
        </div>
    </article>
);

const Testimonials = () => {
  const { t } = useTranslation()
  const testimonialsTranslation = t('testimonials.items', { returnObjects: true })
  const testimonialsData = Array.isArray(testimonialsTranslation) ? testimonialsTranslation : []

  return (
    <section className="py-spacing-3xl bg-neutral-gray-100">
      <div className="container mx-auto px-spacing-md">
        <h2 className="text-font-size-h2 text-center text-neutral-dark mb-spacing-2xl font-extrabold">
          {t('testimonials.title')}
        </h2>
        <div className="grid grid-cols-1 md:grid-cols-3 gap-spacing-xl">
          {testimonialsData.map((testimonial, index) => (
            <TestimonialCard key={index} testimonial={testimonial} />
          ))}
        </div>
      </div>
    </section>
  )
}

export default Testimonials
