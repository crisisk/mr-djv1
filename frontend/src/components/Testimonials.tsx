export interface TestimonialEntry {
  quote: string;
  author: string;
  source: string;
  rating: number;
}

export interface TestimonialsProps {
  testimonials?: TestimonialEntry[];
}

const defaultTestimonials: TestimonialEntry[] = [
  {
    quote: 'De combinatie van DJ en Saxofonist was het hoogtepunt van onze bruiloft. De sfeer was onvergetelijk!',
    author: 'Jan & Marieke',
    source: 'Bruiloft, Amsterdam',
    rating: 5,
  },
  {
    quote: 'Professioneel, energiek en ze wisten precies de juiste snaar te raken. Een absolute aanrader voor elk bedrijfsfeest.',
    author: 'Suzanne van Dijk',
    source: 'Bedrijfsfeest, Utrecht',
    rating: 5,
  },
  {
    quote: 'De muziek was perfect afgestemd op onze gasten. De saxofonist maakte het helemaal af!',
    author: 'Mark de Vries',
    source: 'Verjaardagsfeest, Eindhoven',
    rating: 5,
  },
];

function StarRating({ rating }: { rating: number }) {
  return (
    <div className="flex">
      {Array.from({ length: 5 }).map((_, index) => (
        <svg
          key={index}
          className={`w-5 h-5 ${index < rating ? 'text-secondary' : 'text-neutral-gray-500'}`}
          fill="currentColor"
          viewBox="0 0 20 20"
          xmlns="http://www.w3.org/2000/svg"
          aria-hidden="true"
        >
          <path d="M9.049 2.927c.3-.921 1.603-.921 1.902 0l1.07 3.292a1 1 0 00.95.69h3.462c.969 0 1.371 1.24.588 1.81l-2.8 2.034a1 1 0 00-.364 1.118l1.07 3.292c.3.921-.755 1.688-1.54 1.118l-2.8-2.034a1 1 0 00-1.175 0l-2.8 2.034c-.784.57-1.838-.197-1.539-1.118l1.07-3.292a1 1 0 00-.364-1.118l-2.8-2.034c-.783-.57-.38-1.81.588-1.81h3.461a1 1 0 00.951-.69l1.07-3.292z" />
        </svg>
      ))}
    </div>
  );
}

function TestimonialCard({ testimonial }: { testimonial: TestimonialEntry }) {
  return (
    <article className="bg-neutral-light p-spacing-xl rounded-lg shadow-xl flex flex-col h-full" aria-label={`Testimonial van ${testimonial.author}`}>
      <StarRating rating={testimonial.rating} />
      <p className="lead text-neutral-dark text-emphasis my-spacing-lg flex-grow">“{testimonial.quote}”</p>
      <footer className="border-t border-neutral-gray-100 pt-spacing-md">
        <p className="body-md text-strong text-primary">{testimonial.author}</p>
        <p className="body-sm text-neutral-gray-500">{testimonial.source}</p>
      </footer>
    </article>
  );
}

export function Testimonials({ testimonials = defaultTestimonials }: TestimonialsProps) {
  return (
    <section className="py-spacing-3xl bg-neutral-gray-100">
      <div className="container mx-auto px-spacing-md">
        <h2 className="heading-2 text-center text-neutral-dark mb-spacing-2xl">Wat Klanten Zeggen</h2>
        <div className="grid grid-cols-1 md:grid-cols-3 gap-spacing-xl">
          {testimonials.map((testimonial) => (
            <TestimonialCard key={`${testimonial.author}-${testimonial.source}`} testimonial={testimonial} />
          ))}
        </div>
      </div>
    </section>
  );
}

export default Testimonials;
export { defaultTestimonials };
