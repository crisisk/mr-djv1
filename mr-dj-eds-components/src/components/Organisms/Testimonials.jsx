import React, { useEffect, useMemo } from 'react';
import { trackEvent } from '../../lib/analytics.js';

const testimonialsData = [
  {
    id: 'bruiloft-eindhoven-2025',
    quote: 'Van ceremonie tot laatste dans perfect begeleid. Onze gasten stonden tot het einde op de dansvloer.',
    author: 'Sarah & Tom',
    source: 'Bruiloft • Eindhoven • juni 2025',
    rating: 5,
    category: 'bruiloft',
    media: 'Bruiloft Eindhoven',
  },
  {
    id: 'bruiloft-tilburg-2024',
    quote: 'De saxofonist maakte onze openingsdans magisch. Alles liep strak volgens draaiboek.',
    author: 'Lotte & Wouter',
    source: 'Bruiloft • Tilburg • april 2024',
    rating: 5,
    category: 'bruiloft',
    media: 'Bruiloft Tilburg',
  },
  {
    id: 'bruiloft-maastricht-2025',
    quote: 'Ceremonie audio, vocalist en DJ: alles liep vlekkeloos in Château Neercanne.',
    author: 'Amber & Jelle',
    source: 'Bruiloft • Maastricht • september 2025',
    rating: 5,
    category: 'bruiloft',
    media: 'Bruiloft Maastricht',
  },
  {
    id: 'bruiloft-rotterdam-2024',
    quote: 'Marokkaanse percussion gecombineerd met club classics: perfecte mix voor onze gasten.',
    author: 'Noor & Ilias',
    source: 'Bruiloft • Rotterdam • juli 2024',
    rating: 5,
    category: 'bruiloft',
    media: 'Bruiloft Rotterdam',
  },
  {
    id: 'corporate-eindhoven-2024',
    quote: 'Branding visuals, strakke cues en een energieke host. Onze awardshow voelde premium.',
    author: 'Philips Health',
    source: 'Bedrijfsfeest • Eindhoven • november 2024',
    rating: 5,
    category: 'bedrijfsfeest',
    media: 'Corporate Eindhoven',
  },
  {
    id: 'corporate-denbosch-2025',
    quote: 'Interactieve DJ + MC, perfecte mix voor 500 medewerkers. Survey score: 97% tevredenheid.',
    author: 'Heineken Nederland',
    source: 'Bedrijfsfeest • Den Bosch • februari 2025',
    rating: 5,
    category: 'bedrijfsfeest',
    media: 'Corporate Den Bosch',
  },
  {
    id: 'corporate-tilburg-2023',
    quote: 'Van borrel tot afterparty perfect getimed. De CO₂-jets waren de kers op de taart.',
    author: 'CZ Zorgverzekeringen',
    source: 'Bedrijfsfeest • Tilburg • december 2023',
    rating: 5,
    category: 'bedrijfsfeest',
    media: 'Corporate Tilburg',
  },
  {
    id: 'corporate-amsterdam-2025',
    quote: 'Hybride event met internationale gasten. Streaming audio en tweetalige MC werkten naadloos.',
    author: 'Booking.com',
    source: 'Bedrijfsfeest • Amsterdam • mei 2025',
    rating: 5,
    category: 'bedrijfsfeest',
    media: 'Corporate Amsterdam',
  },
  {
    id: 'private-veldhoven-2024',
    quote: 'Van seventies classics tot hedendaagse hits, iedereen danste – zelfs oma van 82!',
    author: 'Familie Vermeer',
    source: 'Jubileum • Veldhoven • augustus 2024',
    rating: 5,
    category: 'private',
    media: 'Jubileum Veldhoven',
  },
  {
    id: 'private-breda-2023',
    quote: 'Silent disco + live visuals in de tuin. Binnen 90 minuten opgebouwd – legendarische avond.',
    author: "Lisa's 30th",
    source: 'Verjaardag • Breda • september 2023',
    rating: 5,
    category: 'private',
    media: 'Verjaardag Breda',
  },
];

const StarRating = ({ rating }) => {
  const stars = Array(5)
    .fill(0)
    .map((_, i) => (
      <svg
        key={i}
        className={`w-5 h-5 ${i < rating ? 'text-secondary' : 'text-neutral-gray-500'}`}
        fill="currentColor"
        viewBox="0 0 20 20"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path d="M9.049 2.927c.3-.921 1.603-.921 1.902 0l1.07 3.292a1 1 0 00.95.69h3.462c.969 0 1.371 1.24.588 1.81l-2.8 2.034a1 1 0 00-.364 1.118l1.07 3.292c.3.921-.755 1.688-1.54 1.118l-2.8-2.034a1 1 0 00-1.175 0l-2.8 2.034c-.784.57-1.838-.197-1.539-1.118l1.07-3.292a1 1 0 00-.364-1.118l-2.8-2.034c-.783-.57-.38-1.81.588-1.81h3.461a1 1 0 00.951-.69l1.07-3.292z" />
      </svg>
    ));
  return <div className="flex">{stars}</div>;
};

const categories = [
  {
    key: 'bruiloft',
    title: 'Bruiloften 2023–2025',
    description: 'Recente bruiloften waar 100% dansgarantie is waargemaakt en upsells zoals sax of sparkulars zijn ingezet.',
  },
  {
    key: 'bedrijfsfeest',
    title: 'Bedrijfsfeesten met impact',
    description: 'Corporate events met branding, MC en meetbare tevredenheidsscores boven 95%.',
  },
];

const useCategorisedTestimonials = (segment) =>
  useMemo(() => {
    if (!segment) {
      return categories.map((category) => ({
        ...category,
        testimonials: testimonialsData.filter((item) => item.category === category.key).slice(0, 3),
      }));
    }

    const testimonialIndex = testimonialsData.reduce((acc, testimonial) => {
      acc[testimonial.id] = testimonial;
      return acc;
    }, {});

    const highlightIds = Array.isArray(segment.testimonialIds) ? segment.testimonialIds : [];
    const highlightTestimonials = highlightIds
      .map((id) => testimonialIndex[id])
      .filter(Boolean);

    const highlightCategory = segment.highlightCategory || highlightTestimonials[0]?.category || 'bruiloft';
    const baseCategory = categories.find((category) => category.key === highlightCategory);

    const primaryGroup = {
      key: 'personalization-highlight',
      title: segment.headline || baseCategory?.title || 'Uitgelichte cases',
      description: segment.subheadline || baseCategory?.description || '',
      testimonials:
        highlightTestimonials.length > 0
          ? highlightTestimonials
          : testimonialsData.filter((item) => item.category === highlightCategory).slice(0, 3),
    };

    const remainingGroups = categories
      .filter((category) => category.key !== highlightCategory)
      .map((category) => ({
        ...category,
        testimonials: testimonialsData.filter((item) => item.category === category.key).slice(0, 3),
      }));

    return [primaryGroup, ...remainingGroups];
  }, [segment]);

const TestimonialCard = ({ testimonial, segmentId }) => (
  <article className="flex flex-col gap-spacing-sm rounded-3xl border border-neutral-gray-100 bg-neutral-light p-spacing-xl shadow-lg">
    <StarRating rating={testimonial.rating} />
    <p className="text-font-size-h3 text-neutral-dark italic flex-grow">“{testimonial.quote}”</p>
    <div className="border-t border-neutral-gray-100 pt-spacing-sm">
      <p className="text-font-size-body font-bold text-primary">{testimonial.author}</p>
      <p className="text-font-size-small text-neutral-gray-500">{testimonial.source}</p>
    </div>
    <button
      type="button"
      className="self-start text-sm font-semibold text-primary underline"
      onClick={() =>
        trackEvent('testimonial_cta_click', {
          testimonial_id: testimonial.id,
          category: testimonial.category,
          personalization_variant: segmentId || 'default',
        })
      }
    >
      Bekijk media
    </button>
  </article>
);

const MediaTile = ({ label }) => (
  <div className="rounded-3xl border border-neutral-gray-100 bg-gradient-to-br from-primary/15 to-secondary/20 p-spacing-xl text-center text-sm font-semibold text-neutral-dark shadow-md">
    {label}
  </div>
);

const Testimonials = ({ segment }) => {
  const groupedTestimonials = useCategorisedTestimonials(segment);
  const mediaTiles = Array.isArray(segment?.mediaTiles) && segment.mediaTiles.length
    ? segment.mediaTiles
    : testimonialsData.slice(0, 6).map((testimonial) => testimonial.media);

  useEffect(() => {
    groupedTestimonials.forEach((group) => {
      group.testimonials.forEach((testimonial) => {
        trackEvent('testimonial_impression', {
          testimonial_id: testimonial.id,
          category: testimonial.category,
          placement: 'design_system',
          personalization_variant: segment?.id || 'default',
        });
      });
    });
  }, [groupedTestimonials, segment?.id]);

  return (
    <section className="py-spacing-3xl bg-neutral-gray-100">
      <div className="container mx-auto px-spacing-md">
        <h2 className="text-font-size-h2 text-center text-neutral-dark mb-spacing-xl font-extrabold">
          {segment?.headline || 'Social Proof & Media'}
        </h2>
        <p className="text-center text-neutral-gray-600 max-w-2xl mx-auto mb-spacing-2xl">
          {segment?.subheadline ||
            'Toon direct bewijs met categorie-specifieke testimonials en koppel ze aan je mediagallery. DataLayer events volgen impressies en kliks voor CRO-analyse.'}
        </p>
        <div className="grid gap-spacing-xl md:grid-cols-2">
          {groupedTestimonials.map((group) => (
            <div key={group.key} className="space-y-spacing-md">
              <header>
                <h3 className="text-font-size-h3 font-semibold text-neutral-dark">{group.title}</h3>
                <p className="text-font-size-small text-neutral-gray-600">{group.description}</p>
              </header>
              <div className="space-y-spacing-md">
                {group.testimonials.map((testimonial) => (
                  <TestimonialCard key={testimonial.id} testimonial={testimonial} segmentId={segment?.id} />
                ))}
              </div>
            </div>
          ))}
          <div className="space-y-spacing-md">
            <h3 className="text-font-size-h3 font-semibold text-neutral-dark">Media gallery koppeling</h3>
            <p className="text-font-size-small text-neutral-gray-600">
              Selecteer bij elke testimonial het corresponderende beeldmateriaal zodat bezoekers de sfeer direct zien.
            </p>
            <div className="grid grid-cols-2 gap-spacing-md">
              {mediaTiles.map((label) => (
                <MediaTile key={label} label={label} />
              ))}
            </div>
          </div>
        </div>
      </div>
    </section>
  );
};

export default Testimonials;
