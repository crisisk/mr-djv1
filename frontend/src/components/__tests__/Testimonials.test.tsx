import { render } from '@testing-library/react';
import { Testimonials, type TestimonialEntry } from '../Testimonials';

describe('Testimonials', () => {
  it('matches the snapshot for winter events', () => {
    const testimonials: TestimonialEntry[] = [
      {
        quote: 'Een warme sfeer ondanks de kou buiten. De DJ heeft iedereen aan het dansen gekregen.',
        author: 'Elise & Bram',
        source: 'Winterfeest, Breda',
        rating: 5,
      },
      {
        quote: 'Onze klanten waren onder de indruk van de professionele uitstraling en muzikale begeleiding.',
        author: 'Mila Jansen',
        source: 'Zakelijk event, Den Haag',
        rating: 4,
      },
    ];

    const { container } = render(<Testimonials testimonials={testimonials} />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
