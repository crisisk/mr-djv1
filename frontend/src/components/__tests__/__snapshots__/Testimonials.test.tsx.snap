// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testimonials matches the snapshot for winter events 1`] = `
<section
  class="py-spacing-3xl bg-neutral-gray-100"
>
  <div
    class="container mx-auto px-spacing-md"
  >
    <h2
      class="text-font-size-h2 text-center text-neutral-dark mb-spacing-2xl font-extrabold"
    >
      Wat Klanten Zeggen
    </h2>
    <div
      class="grid grid-cols-1 md:grid-cols-3 gap-spacing-xl"
    >
      <article
        aria-label="Testimonial van Elise & Bram"
        class="bg-neutral-light p-spacing-xl rounded-lg shadow-xl flex flex-col h-full"
      >
        <div
          class="flex"
        >
          <svg
            aria-hidden="true"
            class="w-5 h-5 text-secondary"
            fill="currentColor"
            viewBox="0 0 20 20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M9.049 2.927c.3-.921 1.603-.921 1.902 0l1.07 3.292a1 1 0 00.95.69h3.462c.969 0 1.371 1.24.588 1.81l-2.8 2.034a1 1 0 00-.364 1.118l1.07 3.292c.3.921-.755 1.688-1.54 1.118l-2.8-2.034a1 1 0 00-1.175 0l-2.8 2.034c-.784.57-1.838-.197-1.539-1.118l1.07-3.292a1 1 0 00-.364-1.118l-2.8-2.034c-.783-.57-.38-1.81.588-1.81h3.461a1 1 0 00.951-.69l1.07-3.292z"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="w-5 h-5 text-secondary"
            fill="currentColor"
            viewBox="0 0 20 20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M9.049 2.927c.3-.921 1.603-.921 1.902 0l1.07 3.292a1 1 0 00.95.69h3.462c.969 0 1.371 1.24.588 1.81l-2.8 2.034a1 1 0 00-.364 1.118l1.07 3.292c.3.921-.755 1.688-1.54 1.118l-2.8-2.034a1 1 0 00-1.175 0l-2.8 2.034c-.784.57-1.838-.197-1.539-1.118l1.07-3.292a1 1 0 00-.364-1.118l-2.8-2.034c-.783-.57-.38-1.81.588-1.81h3.461a1 1 0 00.951-.69l1.07-3.292z"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="w-5 h-5 text-secondary"
            fill="currentColor"
            viewBox="0 0 20 20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M9.049 2.927c.3-.921 1.603-.921 1.902 0l1.07 3.292a1 1 0 00.95.69h3.462c.969 0 1.371 1.24.588 1.81l-2.8 2.034a1 1 0 00-.364 1.118l1.07 3.292c.3.921-.755 1.688-1.54 1.118l-2.8-2.034a1 1 0 00-1.175 0l-2.8 2.034c-.784.57-1.838-.197-1.539-1.118l1.07-3.292a1 1 0 00-.364-1.118l-2.8-2.034c-.783-.57-.38-1.81.588-1.81h3.461a1 1 0 00.951-.69l1.07-3.292z"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="w-5 h-5 text-secondary"
            fill="currentColor"
            viewBox="0 0 20 20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M9.049 2.927c.3-.921 1.603-.921 1.902 0l1.07 3.292a1 1 0 00.95.69h3.462c.969 0 1.371 1.24.588 1.81l-2.8 2.034a1 1 0 00-.364 1.118l1.07 3.292c.3.921-.755 1.688-1.54 1.118l-2.8-2.034a1 1 0 00-1.175 0l-2.8 2.034c-.784.57-1.838-.197-1.539-1.118l1.07-3.292a1 1 0 00-.364-1.118l-2.8-2.034c-.783-.57-.38-1.81.588-1.81h3.461a1 1 0 00.951-.69l1.07-3.292z"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="w-5 h-5 text-secondary"
            fill="currentColor"
            viewBox="0 0 20 20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M9.049 2.927c.3-.921 1.603-.921 1.902 0l1.07 3.292a1 1 0 00.95.69h3.462c.969 0 1.371 1.24.588 1.81l-2.8 2.034a1 1 0 00-.364 1.118l1.07 3.292c.3.921-.755 1.688-1.54 1.118l-2.8-2.034a1 1 0 00-1.175 0l-2.8 2.034c-.784.57-1.838-.197-1.539-1.118l1.07-3.292a1 1 0 00-.364-1.118l-2.8-2.034c-.783-.57-.38-1.81.588-1.81h3.461a1 1 0 00.951-.69l1.07-3.292z"
            />
          </svg>
        </div>
        <p
          class="text-font-size-h3 text-neutral-dark italic my-spacing-lg flex-grow"
        >
          “
          Een warme sfeer ondanks de kou buiten. De DJ heeft iedereen aan het dansen gekregen.
          ”
        </p>
        <footer
          class="border-t border-neutral-gray-100 pt-spacing-md"
        >
          <p
            class="text-font-size-body font-bold text-primary"
          >
            Elise & Bram
          </p>
          <p
            class="text-font-size-small text-neutral-gray-500"
          >
            Winterfeest, Breda
          </p>
        </footer>
      </article>
      <article
        aria-label="Testimonial van Mila Jansen"
        class="bg-neutral-light p-spacing-xl rounded-lg shadow-xl flex flex-col h-full"
      >
        <div
          class="flex"
        >
          <svg
            aria-hidden="true"
            class="w-5 h-5 text-secondary"
            fill="currentColor"
            viewBox="0 0 20 20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M9.049 2.927c.3-.921 1.603-.921 1.902 0l1.07 3.292a1 1 0 00.95.69h3.462c.969 0 1.371 1.24.588 1.81l-2.8 2.034a1 1 0 00-.364 1.118l1.07 3.292c.3.921-.755 1.688-1.54 1.118l-2.8-2.034a1 1 0 00-1.175 0l-2.8 2.034c-.784.57-1.838-.197-1.539-1.118l1.07-3.292a1 1 0 00-.364-1.118l-2.8-2.034c-.783-.57-.38-1.81.588-1.81h3.461a1 1 0 00.951-.69l1.07-3.292z"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="w-5 h-5 text-secondary"
            fill="currentColor"
            viewBox="0 0 20 20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M9.049 2.927c.3-.921 1.603-.921 1.902 0l1.07 3.292a1 1 0 00.95.69h3.462c.969 0 1.371 1.24.588 1.81l-2.8 2.034a1 1 0 00-.364 1.118l1.07 3.292c.3.921-.755 1.688-1.54 1.118l-2.8-2.034a1 1 0 00-1.175 0l-2.8 2.034c-.784.57-1.838-.197-1.539-1.118l1.07-3.292a1 1 0 00-.364-1.118l-2.8-2.034c-.783-.57-.38-1.81.588-1.81h3.461a1 1 0 00.951-.69l1.07-3.292z"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="w-5 h-5 text-secondary"
            fill="currentColor"
            viewBox="0 0 20 20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M9.049 2.927c.3-.921 1.603-.921 1.902 0l1.07 3.292a1 1 0 00.95.69h3.462c.969 0 1.371 1.24.588 1.81l-2.8 2.034a1 1 0 00-.364 1.118l1.07 3.292c.3.921-.755 1.688-1.54 1.118l-2.8-2.034a1 1 0 00-1.175 0l-2.8 2.034c-.784.57-1.838-.197-1.539-1.118l1.07-3.292a1 1 0 00-.364-1.118l-2.8-2.034c-.783-.57-.38-1.81.588-1.81h3.461a1 1 0 00.951-.69l1.07-3.292z"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="w-5 h-5 text-secondary"
            fill="currentColor"
            viewBox="0 0 20 20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M9.049 2.927c.3-.921 1.603-.921 1.902 0l1.07 3.292a1 1 0 00.95.69h3.462c.969 0 1.371 1.24.588 1.81l-2.8 2.034a1 1 0 00-.364 1.118l1.07 3.292c.3.921-.755 1.688-1.54 1.118l-2.8-2.034a1 1 0 00-1.175 0l-2.8 2.034c-.784.57-1.838-.197-1.539-1.118l1.07-3.292a1 1 0 00-.364-1.118l-2.8-2.034c-.783-.57-.38-1.81.588-1.81h3.461a1 1 0 00.951-.69l1.07-3.292z"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="w-5 h-5 text-neutral-gray-500"
            fill="currentColor"
            viewBox="0 0 20 20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M9.049 2.927c.3-.921 1.603-.921 1.902 0l1.07 3.292a1 1 0 00.95.69h3.462c.969 0 1.371 1.24.588 1.81l-2.8 2.034a1 1 0 00-.364 1.118l1.07 3.292c.3.921-.755 1.688-1.54 1.118l-2.8-2.034a1 1 0 00-1.175 0l-2.8 2.034c-.784.57-1.838-.197-1.539-1.118l1.07-3.292a1 1 0 00-.364-1.118l-2.8-2.034c-.783-.57-.38-1.81.588-1.81h3.461a1 1 0 00.951-.69l1.07-3.292z"
            />
          </svg>
        </div>
        <p
          class="text-font-size-h3 text-neutral-dark italic my-spacing-lg flex-grow"
        >
          “
          Onze klanten waren onder de indruk van de professionele uitstraling en muzikale begeleiding.
          ”
        </p>
        <footer
          class="border-t border-neutral-gray-100 pt-spacing-md"
        >
          <p
            class="text-font-size-body font-bold text-primary"
          >
            Mila Jansen
          </p>
          <p
            class="text-font-size-small text-neutral-gray-500"
          >
            Zakelijk event, Den Haag
          </p>
        </footer>
      </article>
    </div>
  </div>
</section>
`;
